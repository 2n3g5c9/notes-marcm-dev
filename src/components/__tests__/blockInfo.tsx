import React from 'react'
import renderer from 'react-test-renderer'

import { BlockInfo } from '@components'

describe('BlockInfo', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<BlockInfo />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
