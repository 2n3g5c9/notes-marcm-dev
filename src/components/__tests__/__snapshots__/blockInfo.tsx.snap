// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockInfo renders correctly 1`] = `
<div
  className="bg-blue-50 border-blue-300 border-l-2 sm:border-l-4 font-sans inline-block my-2 px-4 py-3 relative text-xs sm:text-sm sm:w-auto w-full"
>
  ℹ️ 
</div>
`;
